interface/controller/pokemon_controller.go

```diff
@@ -24,10 +25,20 @@ func NewPokemonController(pin interactor.PokemonInteractor) PokemonController {
 func (pc *pokemonController) GetPokemons(c Context) error {
 	var pokemons []*model.Pokemon
 
-	pokemons, err := pc.pokemonInteractor.GET(pokemons)
+	pokemons, err := pc.pokemonInteractor.Get(pokemons)
 	if err != nil {
 		return err
 	}
 
 	return c.JSON(http.StatusOK, pokemons)
 }
+
+func (pc *pokemonController) SyncPokemons(c Context) error {
```

Comment:
Add a description for this exported function

Reply:
Done